### DISCUSSION TITLE
Pro plan required for trial application

### DISCUSSION THREAD

**Discussion by sy-records** [vercel/community#7096] [Help] [answered]
### Summary

I would like to try [Logalert](https://vercel.com/integrations/logalert) but it requires a pro plan, I am currently on the Hobby plan, is there any way to try it?

The open source plan for vercel has been suspended.

### Steps to Reproduce

<img width="436" alt="image" src="https://github.com/vercel/community/assets/33931153/c36114f4-6f40-4d10-a105-9744d49feafd">

**Comment by amyegan** [comment, marked as answer]
Hi @sy-records. The Logalert template is marked as `Pro / Enterprise Only` because it uses [log drains](https://vercel.com/docs/observability/log-drains) which are not available on Hobby plans. You would need to upgrade to be able to use that feature.
